docs/source/software/packagemanagers/conda.md

```diff
@@ -13,7 +13,7 @@
 ### Creating Environments
 
 :::::{note}
-We recommend avoiding building Conda environments in your `/home`, for its space quota. Instead, Use `/work`, which can be requested by PIs for groups in need of space `/work`.
+We recommend avoiding building Conda environments in your `/home`, for its space quota. Instead, Use `/project`, which can be requested by PIs for groups in need of space `/project`.
```

Comment:
The directory in explorer is /projects not /project

please update for every entry on this page.